Assets/RGScripts/RGActions/RGAction_MoveToPosition.cs

```diff
@@ -0,0 +1,72 @@
+/*
+* This file has been automatically generated. Do not modify.
+*/
+
+using System;
+using System.Collections.Generic;
+using RegressionGames;
+using RegressionGames.RGBotConfigs;
+using UnityEngine;
+
+namespace RegressionGames
+{
+    public class RGAction_MoveToPosition : RGAction
+    {
+        public void Start()
+        {
+            AddMethod("MoveToPosition", new Action<float, float, float>(GetComponent<Player>().MoveToPosition));
+        }
+
+        public override string GetActionName()
+        {
+            return "MoveToPosition";
+        }
+
+        public override void StartAction(Dictionary<string, object> input)
+        {
+            string xInput = input["x"].ToString();

```

Comment:
Something we should fix in the generator.  `input["x"]` is possibly null, we should do some validation that the needed keys exist and log a warning if the action doesn't include the needed parameters

Reply:
Updated the file. The logic to generate those error checks were pushed to RGUnityBots